### DISCUSSION TITLE
Double payment

### DISCUSSION THREAD

**Discussion by kite-uhn** [community/community#47502] [Other Feature Feedback, Questions, & Ideas]
I created this account and paid for it. First I paid for an organization account for $44. Then I realized that i have to upgrade my seat's privilege to PRO to be able to accomplish my requirements. That was $48.
So now I'm not sure if my company credit card is charged for $(48+44) or only $48?
Please note that There would be just this one user on this organization and it doesn't seem fair to pay twice for user in one organization. I prefer to downgrade this account to one single PRO user in that case.
Can you make this clear for me? I have to explain this matter to finance department and I'm completely confused now.

**Comment by SkyBlack1225** [comment]
You can use a debit card to pay, and there are teachings on the Internet how to use it for free, you can refer to how to set
________________________________
寄件者: KITE ***@***.***>
寄件日期: 2023年2月15日 上午 03:10
收件者: community/community ***@***.***>
副本: Subscribed ***@***.***>
主旨: [GitHub Community] Double payment (Discussion #47502)


I created this account and paid for it. First I paid for an organization account for $44. Then I realized that i have to upgrade my seat's privilege to PRO to be able to accomplish my requirements. That was $48.
So now I'm not sure if my company credit card is charged for $(48+44) or only $48?
Please note that There would be just this one user on this organization and it doesn't seem fair to pay twice for user in one organization. I prefer to downgrade this account to one single PRO user in that case.
Can you make this clear for me? I have to explain this matter to finance department and I'm completely confused now.

**Comment by kite-uhn** [comment]
I don't have issue with payment. I want to know if I have been charged once for organization and once for seat separately?

**Reply by kolumnin** [reply]
Can you find if your company was already on your organization seat while you applied for a paid membership?

**Comment by SkyBlack1225** [comment]
CIL Disassembler

https://www.microsoft.com/store/productId/9NBLGGH5KB8B
從 Windows 的郵件<https://go.microsoft.com/fwlink/?LinkId=550986>傳送

**Comment by SkyBlack1225** [comment]
https://youtu.be/hIWikOo0Yhw
從 Windows 的郵件<https://go.microsoft.com/fwlink/?LinkId=550986>傳送

**Comment by SkyBlack1225** [comment]
P {margin-top:0;margin-bottom:0;}
